swarm/network/stream/common_test.go

```diff
@@ -102,32 +105,56 @@ func NewStreamerService(ctx *adapters.ServiceContext) (node.Service, error) {
 	var err error
```

Comment:
The changes in this file suggest it's best to wait for this PR to be merged before we refactor the simulations....

Reply:
possibly y